content/blog-post/making-our-api-more-efficient.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "API Efficiency: How Lean Code Cut Energy Use by 99%"
+date: 2024-08-13
+author: "Ben Hodgkiss"
+status: beta
+section: Infrastructure
```

Comment:
```suggestion
section: Planning data
```

This one is misleading, it defines the section of the website a post falls in rather than the team.